components/serializer.rst

```diff
@@ -512,7 +512,7 @@ There are several types of normalizers available:
     calling the constructor during the denormalization process.
 
     Objects are normalized to a map of property names (method name stripped of
-    the "get"/"set"/"has"/"remove" prefix and converted to lower case) to property
```

Comment:
the **first letter* is lower cased (getFoo -> foo)

Reply:
So may I suggest: "Objects are normalized to a map of property names (method name stripped of the "get"/"set"/"has"/"remove" prefix and first letter converted to lower case) ..."